dataretrieval/nwis.py

```diff
@@ -71,6 +75,8 @@ def get_qwdata(datetime_index=True, wide_format=True, sites=None, start=None, en
         If the qwdata parameter begin_date is supplied, it will overwrite the start parameter
     end: string
         If the qwdata parameter end_date is supplied, it will overwrite the end parameter
+    multi_index: boolean
```

Comment:
ah, multi_index is a boolean. Set the default to a boolean instead of None.  That'll clean up the code in the format_response function above.